src/content/hcp/product-landing.json

```diff
@@ -40,58 +40,42 @@
 	"blocks": [
 		{
 			"type": "heading",
-			"heading": "Use Cases",
+			"heading": "Featured Documentation",
 			"level": 2,
 			"size": 400
 		},
 		{
-			"type": "tutorial_cards",
-			"tutorialSlugs": ["vault/transform", "vault/static-secrets"]
+			"type": "collection_cards",
+			"collectionSlugs": ["cloud/networking"]
```

Comment:
Question: should this collection card be under the "Featured Documentation" section since it doesn't link to docs, but tutorials?

![image](https://user-images.githubusercontent.com/43934258/190219696-001fbde6-7750-418d-824d-a42d135f58fb.png)

Reply:
nope! i like how this looks, though we should update the heading to "Featured Content" so its more accurate

Reply:
Updates LGTM! 👍🏻 

![image](https://user-images.githubusercontent.com/43934258/190251363-4f4c508f-b073-48e6-8e01-abc5766a710a.png)